charts/sn-platform/values.yaml

```diff
@@ -934,6 +936,9 @@ broker:
   advertisedPodIP: false
   # For exposing kop service for external access
   advertisedDomain: ""
+  dnsNames:
```

Comment:
Would it be better to make it an empty list by default and put those DNS name examples in comments?

Reply:
sure, will comment it out.